config/locales/en.yml

```diff
@@ -705,6 +705,8 @@ en:
           permalink_tip: "This permalink is used to create the url to your shop: %{link}your-shop-name/shop"
           link_to_front: Link to shop front
           link_to_front_tip: A direct link to your shopfront on the Open Food Network.
+          ofn_uid: OFN UID
+          ofn_uid_tip: The uniq id used to identify the enterprise on Open Food Network.
```

Comment:
In Ruby it's `uniq` but in plain English it's spelled "unique".

Reply:
I guess I am talking too much to machines ;)